src/main/java/seedu/address/logic/commands/AddTaskCommand.java

```diff
@@ -58,13 +65,43 @@ public AddTaskCommand(String taskName, LocalDateTime dateTime, Set<Tag> tags, Li
         this.dateTime = dateTime;
         this.tags = tags;
         this.link = link;
+        this.recurrence = 0;
+        this.period = 0;
+    }
+
+    /**
+     * Constructor for AddTaskCommand. Takes in 7 parameters, taskName, dateTime and tags.
```

Comment:
Constructor seems to take in only 6 parameter.
And maybe can update "Takes in 7 parameters, taskName, dateTime and tags." -> "Takes in 6 parameters, taskName, dateTime, tags, link, recurrence and period."